src/it/java/uk/gov/companieshouse/company_appointments/tests/CompanyAppointmentControllerITest.java

```diff
@@ -175,7 +177,7 @@ void testReturn200OKIfAllOfficersAreFound() throws Exception {
                 .andExpect(MockMvcResultMatchers.jsonPath("$.total_results", is(3)));
     }
     @Test
-    void testReturn404IfOfficersForCompanyIsNotFound() throws Exception {
+    void testReturnBaseResponseIfOfficersForCompanyIsNotFound() throws Exception {
```

Comment:
The name doesn't tell much. How about `shouldReturnZeroCountsWhenOfficerForCompanyIsNotFound()`

Reply:
addressed in most new commit